### PR TITLE
Create Blog “okta-sso-integration-for-green-lake-and-aruba-central”

### PR DESCRIPTION
**Pull request by WifiGuyWill**
Automatically generated by Netlify CMS

### CODE REVIEW

File: content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
+---
+title: Okta SSO integration for Green Lake and Aruba Central
+date: 2023-02-14T20:11:58.846Z
+externalLink: https://blog.wifig-guys.com
```
Comment (choukden) [inline content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md:4]: Hi Will,

Not sure if you want to create a "blog link" that points back to your external and existing blog post in the web site https://blog.wifi-guys,com or if you would like to create a new blog post submission to be published on the HPE Developer Community.

If it is the later, no need to specify the external link in your entry. 
Now, if you want to create a blog link that points the readers back to your original post in your web site, just keep the external link (please make sure to adjust the website URL, there is a typo in it) and there is no need to fill in the "Body". Please refer to guidelines here: 
https://github.com/hpe-dev-incubator/hpe-dev-portal/blob/master/docs/ContributorGuide-v2.md#onboarding-an-external-blog-post

**Review by choukden — CHANGES_REQUESTED** [review]
Hi Will,

Thank you very much for your contribution to the HPE Developer Community.

There is one typo in the URL for your external link.

Also, Not sure if you want to create a "blog link" that connects back to your external and existing blog post in the web site https://blog.wifi-guys,com/ or if you would like to create a new blog post submission to be published on the HPE Developer Community.

If it is the later, no need to specify the external link in your entry. Please remove the external link from your entry. This would then trigger a technical review and an editorial review from our team.

Now, if you want to create a blog link that connects the readers back to your original post in your web site, just keep the external link (please make sure to adjust the website URL, there is a typo in it) and there is no need to fill in the "Body". Please refer to guidelines here for detailed information:
https://github.com/hpe-dev-incubator/hpe-dev-portal/blob/master/docs/ContributorGuide-v2.md#onboarding-an-external-blog-post

Best regards,
Denis

File: content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
+---
+title: Okta SSO integration for Green Lake and Aruba Central
```
Comment (choukden) [inline content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md:2]: Title should be "Okta SSO integration for HPE GreenLake and Aruba Central".

File: content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+   ![](/img/ws-image11.png)
+3. Input the metadata from the above step
+
+   While GreenLake does support manually entering this info, it’s recommended to simply upload the XML metadata that was downloaded in the previous step. Select Metadata File and select the XML. Then click Next.
+
+   ![](/img/ws-image12.png)
+4. Enter the SAML attributes to match what was entered in Okta and set the idle timeout value as well.
+
+   ![](/img/ws-image13.png)
+5. Then click Next.
```
Comment (drensing) [inline content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md:126]: Not sure why this font size is smaller than that for Step 6 below it. Maybe Denis can help here?

File: content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+6. Create a recovery user in the event SSO fails, an admin will still be able to access GreenLake.
```
Comment (drensing) [inline content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md:127]: Rephrase this to:
"Create a recover user so that, in the event SSO fails, an admin will still be able to access HPE GreenLake." (actually, you probably need to add a little more to the end of that... so the admin will still be able to access what in HPE GreenLake or the HPE GreenLake what?)

File: content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+
+   ![](/img/ws-image14.png)
+
+   Congratulations SSO will now be enabled for GreenLake as well as the Aruba Central application. Log out and on the GreenLake home page, click Sign in with SSO.
```
Comment (drensing) [inline content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md:131]: Place an exclamation point after "Congratulations" (or at least a period). Add "HPE" in front of both instances of "GreenLake"

File: content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md [hunk 6]
```diff
@@ -0,0 +1,150 @@
...
+
+**Testing and Troubleshooting:**
```
Comment (drensing) [inline content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md:133]: Use sentence case:
"Testing and troubleshooting"

File: content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md [hunk 7]
```diff
@@ -0,0 +1,150 @@
...
+
+On the GreenLake Cloud Platform home page, Click Sign in with SSO.
```
Comment (drensing) [inline content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md:135]: Add "HPE" proir to "GreenLake". Change "Click" to "click". Change "in" to "In", since that's what's shown in the image. So it should read:
"On the HPE GreenLake Cloud Platform home page, click Sign In with SSO."

File: content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md [hunk 8]
```diff
@@ -0,0 +1,150 @@
...
+
+![](/img/ws-image15.png)
+
+![](/img/ws-image16.png)
+
+Enter the SSO credentials and you will be redirected to Okta to authenticate. Once you successfully authenticate, you will be redirected back to GreenLake. You can then click on the Aruba Central application and given access based on the configured role/permissions.
```
Comment (drensing) [inline content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md:141]: Place a period after "credentials". Start the next sentence with "You will be redirected..."
Add "HPE" prior to "GreenLake" in the second sentence.
Add the word "be" prior to "given" in the last sentence.

File: content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md [hunk 9]
```diff
@@ -0,0 +1,150 @@
...
+
+**Additional Notes:**
```
Comment (drensing) [inline content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md:143]: Sentence case for headers, so it reads:
"Additional notes:"

File: content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md [hunk 10]
```diff
@@ -0,0 +1,150 @@
...
+
+* There must be at least **one** verified user belonging to the **Domain** prior to configuration.
+* In order to configure SSO, you must be logged into GreenLake with a user from the domain.
```
Comment (drensing) [inline content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md:146]: Add "HPE" in front of "GreenLake"

File: content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md [hunk 11]
```diff
@@ -0,0 +1,150 @@
...
+* SSO user access is determined by the “role_name” attribute included in the SAML hpe_ccs_attribute provided by the IdP.
+* SSO users can initiate a Single Sign On request by trying to log in to Aruba Central (SP-initiated login).
```
Comment (drensing) [inline content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md:148]: Change "... log in to..." to "...log into..."

**Review by drensing — COMMENTED** [review]
Hi, Will,
A great blog for us to have. Thank you. I've suggested some paraphrasing so we can do better in regards to the SEO issue - just changing one paragraph, a title and dropping some sections really won't do it. We really need a bit of rephrasing. Also pointed out those areas where HPE GreenLake needs to be properly branded. Other than that, not many grammatical edits.

All the best,
Dale

File: content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md [hunk 12]
```diff
@@ -0,0 +1,150 @@
...
+author: "Will Smith - Consulting Systems Engineer (ACEX #34)"
+authorimage: /img/willsmith-sm.jpg
+thumbnailimage: /img/gl-sso.jpg
+disable: false
+tags:
+  - aruba-central
+  - hpe-greenlake-cloud-platform
+  - sso
+  - okta
+---
```
[the remark this answers is not in the document]
Comment (WifiGuyWill) [reply content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md:14]: Thanks for feedback, much appreciated. Going through the changes/suggestions now.

File: content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md [hunk 13]
```diff
@@ -0,0 +1,150 @@
...
+Aruba Central has gone GREEN…GreenLake that is! If you’re reading this, then you’ve likely heard that Aruba Central is now integrated into the new HPE GreenLake Common Cloud Platform (GLCP). This allows IT admins to view and orchestrate critical network services such as Wired, Wireless and SD-Branch, through the same dashboard as their compute and storage infrastructure.
```
[the remark this answers is not in the document]
Comment (WifiGuyWill) [reply content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md:15]: Updated:
Aruba Central has gone GREEN…GreenLake that is! Aruba Central has recently been integrated into the HPE GreenLake Cloud Platform (GLCP). This provides IT administrators with the ability to view and orchestrate critical network services, such as Wired, Wireless and SD-Branch, through the same dashboard as their compute and storage infrastructure. GLCP also supports Single Sign On (SSO) which helps simplify account management.

If you are new to Aruba Central and are looking to enable SSO, this guide is for you. It will walk you through the process of configuring SSO for HPE GreenLake and Aruba Central using Okta.

File: content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md [hunk 14]
```diff
@@ -0,0 +1,150 @@
...
+
+If you are new to Aruba Central and are looking to enable SSO, this guide is for you. This will walk you through the process of configuring SSO for GreenLake and Aruba Central using Okta.
```
[the remark this answers is not in the document]
Comment (WifiGuyWill) [reply content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md:17]: Updated: 
If you are new to Aruba Central and are looking to enable SSO, this guide is for you. It will walk you through the process of configuring SSO for HPE GreenLake and Aruba Central using Okta.

File: content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md [hunk 15]
```diff
@@ -0,0 +1,150 @@
...
+
+### Before you Begin
```
[the remark this answers is not in the document]
Comment (WifiGuyWill) [reply content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md:19]: Changed capitalization

File: content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md [hunk 16]
```diff
@@ -0,0 +1,150 @@
...
+
+Please review the [GreenLake](https://support.hpe.com/hpesc/public/docDisplay?docId=a00120892en_us) User Guide to understand how the SAML framework works in the context of Common Cloud Services for the Aruba Central application.
```
[the remark this answers is not in the document]
Comment (WifiGuyWill) [reply content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md:21]: Done

File: content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md [hunk 17]
```diff
@@ -0,0 +1,150 @@
...
+
+### Steps to Configure SSO/SAML Application in Okta
```
[the remark this answers is not in the document]
Comment (WifiGuyWill) [reply content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md:23]: Updated:
### Configure SSO/SAML application in Okta

File: content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md [hunk 18]
```diff
@@ -0,0 +1,150 @@
...
+
+To configure application metadata in Okta, complete the following steps:
+
+* Step 1: Create an Okta SAML application
+* Step 2: Configure Sign On settings
+* Step 3: Export the SAML 2.0 IdP metadata
+* Step 4: Configure the SAML connection in GreenLake
+
+**Step 1: Create an Okta SAML Application**
```
[the remark this answers is not in the document]
Comment (WifiGuyWill) [reply content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md:32]: Updated:
To configure application metadata in Okta, complete the following steps:

* Step 1: Create an Okta SAML application
* Step 2: Configure Sign On settings
* Step 3: Export the SAML 2.0 IdP metadata
* Step 4: Configure the SAML connection in GreenLake

**Step 1: Create an Okta SAML application**

File: content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md [hunk 19]
```diff
@@ -0,0 +1,150 @@
...
+
+1. Log in to the Okta administration console.
+2. Click **Applications > Create App Integration.** The Create a new app integration window opens.
+3. Select SAML 2.0 and click Next.
```
[the remark this answers is not in the document]
Comment (WifiGuyWill) [reply content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md:36]: Next is capitalized in the screenshot, I'll leave it that way in the post.

File: content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md [hunk 20]
```diff
@@ -0,0 +1,150 @@
...
+
+* Step 1: Create an Okta SAML application
+* Step 2: Configure Sign On settings
+* Step 3: Export the SAML 2.0 IdP metadata
+* Step 4: Configure the SAML connection in GreenLake
+
+**Step 1: Create an Okta SAML Application**
+
+1. Log in to the Okta administration console.
+2. Click **Applications > Create App Integration.** The Create a new app integration window opens.
```
[the remark this answers is not in the document]
Comment (WifiGuyWill) [reply content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md:35]: Adjusted to match screenshot:
1. Log in to the Okta administration console.
2. Click **Applications > Create new app integration.** The Create a new app integration window opens.

File: content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md [hunk 21]
```diff
@@ -0,0 +1,150 @@
...
+3. Select SAML 2.0 and click Next.
+
+![](/img/ws-image0.png)
+
+Provide a name for the Aruba GreenLake SSO service (Okta Application)
```
[the remark this answers is not in the document]
Comment (WifiGuyWill) [reply content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md]: Provide a name for the Aruba GreenLake SSO service (Okta application)

File: content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md [hunk 22]
```diff
@@ -0,0 +1,150 @@
...
+
+![](/img/ws-image1.png)
+
+**Step 2: Configure Single Sign On settings**
```
[the remark this answers is not in the document]
Comment (WifiGuyWill) [reply content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md:44]: Updated:
**Step 2: How to configure Single Sign On settings**

File: content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md [hunk 23]
```diff
@@ -0,0 +1,150 @@
...
+
+    **gl\_first\_name = user.FirstName**
+
+    **gl\_last\_name = user.LastName**
+
+    **hpe\_ccs\_attribute = (See Below)**
+
+   See here for IdP attribute details: <https://support.hpe.com/hpesc/public/docDisplay?docId=a00120892en_us>
+
+   One of the additional features added as part of the GreenLake platform is Role Based Access Controls (RBAC) for Aruba Central and all other apps in GLCP. A new SAML attribute has been added “hpe_ccs_attribute” which tells GreenLake and Central the exact role/permissions for each user. The following describes how to format the attribute.
```
[the remark this answers is not in the document]
Comment (WifiGuyWill) [reply content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md:68]: Rephrased and fixed the markdown issue with the underscores

File: content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md [hunk 24]
```diff
@@ -0,0 +1,150 @@
...
+
+![](/img/ws-image2.png)
+
+![](/img/ws-image3.png)
+
+![](/img/ws-image4.png)
+
+![](/img/ws-image5.png)
+
+The **hpe\_ccs\_attribute** will always start with **version_1#**. First, we’ll configure the attributes for GreenLake CCS, then Central. First enter the PCID for the account, followed by the GreenLake application ID. This will always be **00000000-0000-0000-0000-000000000000**. Then followed by the role name and **ALL_SCOPES**. Next will be the Aruba Central info. Start with the **app cid**, then the role name (IE Aruba Central Administrator) then **ALL_SCOPES**.
```
[the remark this answers is not in the document]
Comment (WifiGuyWill) [reply content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md:78]: Updated w/ suggested edits:

The **hpe\_ccs\_attribute** always starts with version_1#. You must first configure the attributes for HPE GreenLake CSS, and then Central. To do so, enter the PCID for the account, followed by the HPE GreenLake application ID. This will always be **00000000-0000-0000-0000-000000000000**. Following this, enter the role name and **ALL_SCOPES**. Next, enter in the Aruba Central information. Start with the **app cid**, followed by the role name (i.e. Aruba Central Administrator), and then **ALL_SCOPES**.

File: content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md [hunk 25]
```diff
@@ -0,0 +1,150 @@
...
+
+Example:
+
+**version_1#5b0ec0e8c4f422eca232ba72799953ac:00000000-0000-0000-0000-000000000000:Account Administrator:ALL_SCOPES:683da368-66cb-4ee7-90a9-ec1964768092:**
+
+**Aruba Central Administrator:ALL_SCOPES**
+
+If you want to add additional GreenLake applications or if you have multiple Aruba Central accounts, you can add them as well. Just follow the same syntax as before. Once you have the attribute defined, enter it into the SAML attribute statement in Okta as shown below.
```
[the remark this answers is not in the document]
Comment (WifiGuyWill) [reply content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md:86]: Updated:
If you want to add additional HPE GreenLake applications, or if you have multiple Aruba Central accounts, you can add them as well. Just follow the same syntax as before. Once you have the attribute defined, enter it into the SAML attribute statement in Okta as shown below.

File: content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md [hunk 26]
```diff
@@ -0,0 +1,150 @@
...
+
+![](/img/ws-image6.png)
+
+2﻿. Complete the setup.
+
+![](/img/ws-image7.png)
+
+Click Next and Select “Internal App” then Finish
```
[the remark this answers is not in the document]
Comment (WifiGuyWill) [reply content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md:94]: Updated:
3. Select Internal app, and Click Finish

File: content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md [hunk 27]
```diff
@@ -0,0 +1,150 @@
...
+
+**Step 3:** **Export the SAML 2.0 IdP metadata**
+
+1. Click Next – Configure the Sign On settings
+
+   Two options available: **View Setup Instructions** which steps through the SAML config or **Identity Provider metadata** will produce an XML that can be loaded into Aruba Central.
```
[the remark this answers is not in the document]
Comment (WifiGuyWill) [reply content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md:100]: Updated:
**Step 3:** **Export the SAML 2.0 IdP metadata**

1. Click Next – Configure the Sign On settings

   You will find two options are available: **View Setup Instructions** which steps you through the SAML configuration and **Identity Provider metadata**, which will produce an XML file that can be loaded into Aruba Central.

File: content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md [hunk 28]
```diff
@@ -0,0 +1,150 @@
...
+
+   Suggestion: Click **Identity Provider metadata** and save the XML data to a file.
+
+   ![](/img/ws-image9.png)
+2. C﻿lick Next
+3. Select Internal app and Click Finish
+
+**Step 4: Create SAML Authorization Profile in GreenLake Cloud Platform**
```
[the remark this answers is not in the document]
Comment (WifiGuyWill) [reply content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md:108]: Updated:
**Step 4: Create SAML Authorization Profile in HPE GreenLake Cloud Platform**

File: content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md [hunk 29]
```diff
@@ -0,0 +1,150 @@
...
+
+1. Log into GreenLake and click Menu > Manage > Authentication and Click Set Up SAML Connection.
```
[the remark this answers is not in the document]
Comment (WifiGuyWill) [reply content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md:110]: 1. Log into HPE GreenLake and click Menu > Manage > Authentication and Click Set Up SAML Connection.

File: content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md [hunk 30]
```diff
@@ -0,0 +1,150 @@
...
+
+   *Before you can add a new SAML configuration, you must have at least one user account with that domain already enabled in GreenLake. Also, you must be logged into GreenLake with an account from that domain in order to enable SSO for that domain.*
```
[the remark this answers is not in the document]
Comment (WifiGuyWill) [reply content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md:112]: Updated:
   *Before you can add a new SAML configuration, you must have at least one user account with that domain already enabled in HPE GreenLake. Also, you must be logged into GreenLake with an account from that domain in order to enable SSO for it.*

File: content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md [hunk 31]
```diff
@@ -0,0 +1,150 @@
...
+
+   ![](/img/ws-image10.png)
+2. Type in the domain you want to enable SSO on:
+
+   ![](/img/ws-image11.png)
+3. Input the metadata from the above step
```
[the remark this answers is not in the document]
Comment (WifiGuyWill) [reply content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md:118]: Updated:
3. Input the metadata from the step above.

File: content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md [hunk 32]
```diff
@@ -0,0 +1,150 @@
...
+
+   While GreenLake does support manually entering this info, it’s recommended to simply upload the XML metadata that was downloaded in the previous step. Select Metadata File and select the XML. Then click Next.
```
[the remark this answers is not in the document]
Comment (WifiGuyWill) [reply content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md:120]: Updated:
While HPE GreenLake does support entering this information manually, it's recommended that you simply upload the XML metadata that was downloaded in the previous step. To do so, Select Metadata File, selecting the XML file. Then, click Next.

File: content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md [hunk 33]
```diff
@@ -0,0 +1,150 @@
...
+
+   ![](/img/ws-image12.png)
+4. Enter the SAML attributes to match what was entered in Okta and set the idle timeout value as well.
```
[the remark this answers is not in the document]
Comment (WifiGuyWill) [reply content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md:123]: Updated:
4. Enter the SAML attributes to match what was entered in Okta. Set the idle timeout value as well.

File: content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md (hunk 4 above)
Comment (WifiGuyWill) [reply content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md:127]: Updated:
6. Create a recover user so that, in the event SSO fails, an admin will still be able to access the HPE GreenLake portal.

File: content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md (hunk 5 above)
Comment (WifiGuyWill) [reply content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md:131]: Updated:
   Congratulations! SSO will now be enabled for HPE GreenLake as well as the Aruba Central application. Log out and on the HPE GreenLake home page, click Sign in with SSO.

File: content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md (hunk 6 above)
Comment (WifiGuyWill) [reply content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md:133]: Updated:
**Testing and troubleshooting:**

File: content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md (hunk 7 above)
Comment (WifiGuyWill) [reply content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md:135]: Updated:
On the HPE GreenLake Cloud Platform home page, click Sign In with SSO.

File: content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md (hunk 8 above)
Comment (WifiGuyWill) [reply content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md:141]: Updated:
Enter the SSO credentials. You will be redirected to Okta to authenticate. Once you successfully authenticate, you will be redirected back to HPE GreenLake. You can then click on the Aruba Central application and be given access based on the configured role/permissions.

File: content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md (hunk 9 above)
Comment (WifiGuyWill) [reply content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md:143]: Updated:
**Additional notes:**

File: content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md (hunk 10 above)
Comment (WifiGuyWill) [reply content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md:146]: Updated:
* In order to configure SSO, you must be logged into HPE GreenLake with a user from the domain.

File: content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md (hunk 11 above)
Comment (WifiGuyWill) [reply content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md:148]: Updated:
* SSO users can initiate a Single Sign On request by trying to log into Aruba Central (SP-initiated login).

**Review by WifiGuyWill — COMMENTED** [review]
Changes and suggested updates made.
